tox.ini

```diff
@@ -0,0 +1,16 @@
+[tox]
+envlist = py{38,310,311}-django{32}
+
+
+[testenv]
+envdir=
+    # Use the same environment for all commands running under a specific python version
+    py38: {toxworkdir}/py38
+    py310: {toxworkdir}/py310
+    py311: {toxworkdir}/py311
+
+deps =
+    django32: -r requirements/django.txt
```

Comment:
this file doesn't exist

Reply:
[9adc67b](https://github.com/eduNEXT/eox-nelp/pull/19/commits/9adc67b2646aa508939d14c5134a061b53f93bc0)